examples/cerebro/channel_ops.go

```diff
@@ -98,12 +100,12 @@ func (o *Operator) handleEnableChain(args []string) {
 		return
 	}
 
-	fmt.Printf("Successfully opened custody channel for %s!\n", chainName)
+	fmt.Printf("Successfully opened custody channel on %s!\n", chainName)
 }
 
-func (o *Operator) handleDisableChain(args []string) {
-	if len(args) < 3 {
-		fmt.Println("Usage: disable <chain_name> <token_symbol>")
+func (o *Operator) handleCloseChannel(args []string) {
```

Comment:
add ChannelID (same as previous comments

Reply:
I don't think we need to have the ability to close or smth by ID, my comments more about logs, responses, etc